ClientApp/src/app/shared/team-member.model.ts

```diff
@@ -0,0 +1,18 @@
+export class TeamMember {
```

Comment:
I've been making these types of things in the `ClientApp/types` directory, are you opposed to moving this in there?